component/src/main/java/me/androidbox/component/agenda/AgendaHomeTopBar.kt

```diff
@@ -26,6 +32,7 @@ fun AgendaTopBar(
     onProfileButtonClicked: () -> Unit,
     onDateClicked: () -> Unit,
     modifier: Modifier = Modifier,
+    dropDropMenuScope: @Composable () -> Unit
```

Comment:
@philipplackner Is this the correct way to do this. Any suggestions about the naming?

Reply:
Yes, but the name is a bit weird, why `dropDrop`? What about just `dropDownMenu`?

Reply:
@philipplackner that was a typo error :blush: